src/aave-v3/SupplyVault.sol

```diff
@@ -42,18 +42,18 @@ contract SupplyVault is ISupplyVault, SupplyVaultBase {
     /// STRUCTS ///
 
     struct UserRewardsData {
-        uint128 index; // User rewards index for a given reward token (in wad).
-        uint128 unclaimed; // Unclaimed amount for a given reward token (in reward tokens).
+        uint256 index; // User rewards index for a given reward token (in ray).
+        uint256 unclaimed; // Unclaimed amount for a given reward token (in reward tokens).
     }
 
     /// CONSTANTS AND IMMUTABLES ///
 
-    uint256 public constant SCALE = 1e36;
+    uint256 public constant SCALE = 1e36; // A ray.
```

Comment:
Let's remove it and the value is not changed btw. Also I think we can switch to ray math for multiplication and division

Reply:
Oh, that's stupid of me. I actually thought that a ray was 1e36 the entire time...

Reply:
😆😆 no problem